cli/main.go

```diff
@@ -1 +1,62 @@
 package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/ioki-mobility/go-outline"
+	"github.com/rsjethani/secret/v2"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+
+	var collectionCmd = &cobra.Command{
+		Use:   "collection",
+		Short: "Work with collections",
+		Long:  `If you have to work with collection in any case, use this command`,
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println("Hello World")
+		},
+	}
+
+	var apiKeyFlag string
+	var baseUrlFlag string
+	var collectionExportCmd = &cobra.Command{
+		Use:   "export",
+		Short: "Export a given collection",
+		Long:  "Export a given collection to and safes it in the provided directory",
+		Args:  cobra.MinimumNArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			collectionId := args[0]
+			directoryToSafe := args[1]
+			secretApiKey := secret.NewText(apiKeyFlag)
+			cl := outline.New(baseUrlFlag, secretApiKey)
+			var documents []outline.Document
+			err := cl.
+				Documents().
+				GetAll().
+				Collection(outline.CollectionID(collectionId)).
+				Do(context.Background(), func(d *outline.Document, err error) bool {
+					if err != nil {
+						documents = append(documents, *d)
+					}
+					return true
+				})
+			if err != nil {
+				panic(err)
+			}
+			// TODO: Safe documents in directoryToSafe in markdown file
+			fmt.Println(directoryToSafe)
+			fmt.Printf("%+v", documents)
+		},
+	}
+
+	collectionExportCmd.Flags().StringVar(&baseUrlFlag, "baseUrl", "", "The base url to the outline instance")
+	collectionExportCmd.Flags().StringVar(&apiKeyFlag, "apiKey", "", "The outline apiKey")
```

Comment:
these should be associated with rootCmd

Reply:
Why? This is a sub command of the collection command. Why should it be connected to the rootCmd? 🤔